_posts/2019-01-23-multifile-python-functions.md

```diff
@@ -0,0 +1,213 @@
+---
+title: Multifile python functions
+description: "Tidy python in an OpenFaaS world; or, what happens when you want to split your Python function into multiple modules?"
+date: 2019-01-23
+image: /images/multifile-python-function/pexels-book-stack-books-bookshop-264635.jpg
+categories:
+  - python
+author_staff_member: lucas
+dark_background: true
+---
+
+As a core contributor to OpenFaaS, I hangout in the OpenFaaS Slack a lot, you can [join us here][openfaas-slack-singnup]. Most recently that meant trying to answer this question
```

Comment:
`I hangout in the OpenFaaS Slack a lot, you can [join us here][openfaas-slack-singnup]`

`I hangout in the OpenFaaS Slack a lot (to help new users and contributors?). You can join the community [here][openfaas-slack-singnup]`

Is `singnup` meant to be `signup` ?